### ISSUE TITLE
Consolidate "postprocessing" and "auxiliary" files

### ISSUE THREAD

**Issue by samgdotson** [arfc/mhtgr350-benchmark#56]
Currently, there are multiple files in the repository called ``postprocessing.py`` and ``auxiliary.py``. They have some different functions but serve the same role of processing data into plots and the needed helper functions.

This issue can be closed when there is a single file called ``auxiliary.py`` and ``postprocessing.py`` at the top level of the directory. 

Why? Each of the sub-directories ``fuel-column``, ``full-core``, ``isotope-distribution``, and ``problem-setup`` contain data that is easier to access from the top level of the directory than the other way around -- removing the need for ``pathmagic.py`` or ``sys.path.insert()``. If, in the future, this repository is used to publish results, it will be easier for others to clone the repo and process the data if they don't have to edit the path variables first.

**Comment by robfairh** [comment]
True. I am going to leave this for later cause I have a few more PRs with more auxiliary.py and postprocessing.py.
Once I am done with those PRs, I will work on this issue.
